templates/kubernetes-cluster.template

```diff
@@ -381,6 +383,36 @@ Resources:
             # so pods can communicate
             kubectl apply -f https://${QSS3BucketName}.s3.amazonaws.com/${QSS3KeyPrefix}/scripts/${NetworkingProvider}.yaml
 
+            # Expose the api server over ELB
+            kubectl -n kube-system expose pod \
```

Comment:
yeah testing this it definitely created a new LB, I thought setting kubernetes.io/service-name to the service name would have been enough.

I'm currently trying to see what we can do with the AWS::ElasticLoadBalancingV2::TargetGroup type, which may allow us to create in the following order:

LB -> Master -> TargetGroup

Such that CF is creating all three in order, and the TargetGroup has enough information to go on.  I'm not sure if I'm understanding what a TargetGroup is well enough though, still investigating.